src/pint/models/timing_model.py

```diff
@@ -3037,13 +3037,21 @@ def alias_to_pint_param(self, alias):
                 first_init_par = None
                 # Handle the case of start index from 0 and 1
                 for start_idx in [0, 1]:
-                    first_init_par_alias = prefix + "{1:0{0}}".format(fmt, start_idx)
+                    # Ensure compatibility with tempo-style "DM001"
+                    if "DM0" in alias:
```

Comment:
I don't think  it's a good idea to special-case DM parameters - it will confuse users that DM accepts three digits but DMX (say) doesn't.